README.md

```diff
@@ -101,6 +102,14 @@ mariadb-data
 drunk_goodall
 ```
 
+### Excluding Volumes From Gargbage Collection
+
+There can be occasions where you dont want to remove a dangling volume.
+In such case, you can create `/etc/docker-gc-exclude-volumes` (or specify
```

Comment:
"_To enable this functionality_ you can create _a file named_ `/etc/docker-gc-exclude-volumes`"